### ISSUE TITLE
Body is undefined while running local

### ISSUE THREAD

**Issue by mrpatiwi** [dougmoscrop/serverless-http#4]
Can't be run locally, perhaps `event` is empty

```sh
serverless  invoke local --function handler --log
```

```txt
Type Error ---------------------------------------------

     Cannot read property 'body' of undefined
```

**Comment by mrpatiwi** [comment]
Using a `event.json` like:

```json
{
	"resource": "/users",
	"path": "/users",
	"httpMethod": "GET",
	"headers": {
		"Accept": "*/*",
		"CloudFront-Forwarded-Proto": "https",
		"CloudFront-Is-Desktop-Viewer": "true",
		"CloudFront-Is-Mobile-Viewer": "false",
		"CloudFront-Is-SmartTV-Viewer": "false",
		"CloudFront-Is-Tablet-Viewer": "false",
		"CloudFront-Viewer-Country": "CL",
		"Host": "guku9kxkik.execute-api.us-east-1.amazonaws.com",
		"User-Agent": "curl/7.43.0",
		"Via": "1.1 5c758cc7e759f26aa3968ec4acc2e892.cloudfront.net (CloudFront)",
		"X-Amz-Cf-Id": "LuNtBv-r5oI67cERfiRYbZtpius7XY0EfJ9P8PSradIle_0uvJtrGw==",
		"X-Forwarded-For": "190.196.209.98, 54.239.182.8",
		"X-Forwarded-Port": "443",
		"X-Forwarded-Proto": "https"
	},
	"queryStringParameters": null,
	"pathParameters": null,
	"stageVariables": null,
	"body": null,
	"isBase64Encoded": false
}
```

and running:

```sh
serverless  invoke local --function handler --log --path event.json
```

Works well

**Comment by dougmoscrop** [comment]
This would be by design I think - I'm open to suggestions on how you would like this to work; this is sort of a problem with serverless invoke, which is why there is a discussion to make like `serverless invoke http` because a regular lambda invoke and an API Gateway invoke are quite different

**Comment by dougmoscrop** [comment]
Would you accept that if `event` is undefined, it defaults to:

```json
{
  "method": "GET",
  "path": "/",
  "body": ""
}
```

?

**Comment by mrpatiwi** [comment]
Looks like a razonable default 👍

**Comment by dougmoscrop** [comment]
Released v1.1.0 with this in it
